flag.go

```diff
@@ -3,7 +3,7 @@
 package iproto
 
 // IPROTO flag constants, generated from
-// tarantool/src/box/iproto_constants.h
+// tarantool-master/src/box/iproto_constants.h
```

Comment:
It looks like an unnecessary change when we will regenerate the file for a next tag.
```suggestion
// src/box/iproto_constants.h
```